pkg/cosign/verify.go

```diff
@@ -230,7 +286,7 @@ func VerifyImageSignatures(ctx context.Context, signedImgRef name.Reference, co
 					return err
 				}
 				if cert == nil {
-					return errors.New("no certificate found on signature")
+					return errors.New("no certificate found on sigRef")
```

Comment:
Suggestion: `signature` sounds correct to me, it refers to the command argument name, imho.

Reply:
thank you @hectorj2f, done.